src/taco-cli/test/create.ts

```diff
@@ -47,7 +46,7 @@ import TemplateManager = require ("../cli/utils/templateManager");
 import tacoTestsUtils = require ("taco-tests-utils");
 
 import IKeyValuePair = tacoTestUtils.IKeyValuePair;
-import TacoKitsErrorCodes = tacoKits.TacoErrorCode;
```

Comment:
How did this work before when taco-cli did not take any dependency on taco-kits in package.json?

Oh, it's because our folder structure had things in a root node_modules folder, so it just happened to work.